### DISCUSSION TITLE
Change the Filter Name to the Filter Label

### DISCUSSION THREAD

**Discussion by iAmKevinMcKee** [rappasoft/laravel-livewire-tables#355] [Feature Requests]
I am loving this package. I just added a filter and noticed that the "Applied Filters" section uses the key 'type' in stead of the label "Property Type". I would have expected it to use the label, which in my case is "Property Type". If you think this is a good change I'm happy to try to submit a PR. Thanks.

```php
return [
    'type' => Filter::make('Property Type')
        ->select($propertyTypes),
];
```

Expected this label to be "Property Type" instead of "Type".

![image](https://user-images.githubusercontent.com/4503765/122704844-2106ab80-d21a-11eb-8e02-062ff3ab0338.png)

**Comment by rappasoft** [comment]
Not sure why it chose type on that one, but you can customize the names if it doesn't give you something you like: https://github.com/rappasoft/laravel-livewire-tables/wiki/Customizing-the-filter-names

**Reply by iAmKevinMcKee** [reply]
Thanks, looks great. Appreciate it so much!

**Comment by lrljoe** [comment]
Possible in v2+
